sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clientTelemetry/AzureVMMetadata.java

```diff
@@ -3,7 +3,9 @@
 package com.azure.cosmos.implementation.clientTelemetry;
 
 import com.azure.cosmos.implementation.apachecommons.lang.StringUtils;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
why `JsonIgnoreProperties` is needed?

Reply:
Because Azure Metadata team iterates on the contract and has added some new json nodes (custom in this case) - don't want to be broken every time the metadata team is making (what they would consider non-breaking) changes adding new json nodes.